### PR TITLE
[SKIP CI] Update tenancy.md

### LINKED ISSUE #1536: tenancy.md is outdated and missing out information regarding _VM_DS and _ALL_DS
https://github.com/vmware/docker-volume-vsphere/blob/master/docs/features/tenancy.md

Above file content is really *old* and missing out useful information. It hasn't updated since 03/30 and out of sync with Admin_cli.md.

- [ ] Need to address comments made at #1535

/CC @tusharnt @msterin

### PR DESCRIPTION
**Pull request by lipingxue**
Fixed #1536 
This PR updates tenancy.md to match the up-to-date multi-tenancy behavior.

### CODE REVIEW

File: docs/features/tenancy.md [hunk 1]
```diff
@@ -83,39 +97,40 @@ f15c1f6d-5df5-4a00-8f20-77c8e7a7af11  Product1Test
 Lets limit dev team to create volumes of total 20 Gb each not exceeding size of 1 GB. Similarly for QA team, lets put restriction of total 40 GB consumption with each volume size not exceeding 1 GB.
 
 ```
-#/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=Product1Dev --datastore=datastore3  --allow-create --default-datastore --volume-maxsize=1GB --volume-totalsize=20GB
-vmgroup access add succeeded
+[root@sc-rdops-vm17-dhcp-3-236:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access set --name=Product1Dev --datastore=sharedVmfs-0  --volume-maxsize=1GB --volume-totalsize=20GB
```
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: `--datastore=sharedVmfs-0 ` why? so far the document was referring to `datastore3 `.

File: docs/features/tenancy.md [hunk 2]
```diff
@@ -83,39 +97,40 @@ f15c1f6d-5df5-4a00-8f20-77c8e7a7af11  Product1Test
...
+vmgroup access set succeeded
+
+[root@sc-rdops-vm17-dhcp-3-236:~] /usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access set --name=Product1Test --datastore=sharedVmfs-0  --volume-maxsize=1GB --volume-totalsize=40GB
```
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: same as above

File: docs/features/tenancy.md [hunk 3]
```diff
@@ -83,39 +97,40 @@ f15c1f6d-5df5-4a00-8f20-77c8e7a7af11  Product1Test
...
+vmgroup access set succeeded
 
-#/usr/lib/vmware/vmdkops/bin/vmdkops_admin.py vmgroup access add --name=Product1Test --datastore=datastore3  --allow-create --default-datastore --volume-maxsize=1GB --volume-totalsize=40GB
-vmgroup access add succeeded
 ```
 Lets verify that storage restrictions has been set properly.
```
Comment (shuklanirdesh82) [inline docs/features/tenancy.md]: not for this review, as you are here to update .. can you address following nit?

`restrictions has been set properly` => `restrictions have been set properly`

**Comment by shuklanirdesh82** [comment]
@lipingxue Can you please fix following broke link too from tenancy.md?

` Vmgroups can be created and managed via the [Admin CLI](/user-guide/admin-cli/#Vmgroup)`

File: docs/features/tenancy.md (hunk 1 above)
Comment (lipingxue) [inline docs/features/tenancy.md]: Done.

File: docs/features/tenancy.md (hunk 2 above)
Comment (lipingxue) [inline docs/features/tenancy.md]: Done.

File: docs/features/tenancy.md (hunk 3 above)
Comment (lipingxue) [inline docs/features/tenancy.md]: Done.

**Comment by lipingxue** [comment]
@shuklanirdesh82 I have addressed your comments, please take a look.

File: docs/features/tenancy.md [hunk 4]
```diff
@@ -26,42 +26,56 @@ Vmgroups can be created and managed via the [Admin CLI](/user-guide/admin-cli/#V
...
 ```
 
-### Default privileges
-When a VM tries to manipulate volumes on a datastore which is not configured, the system will use _DEFAULT privilege, which is created automatically by system post install. This _DEFAULT privilege allows access to ANY datastore by default. An admin can edit or remove this record, thus locking down the functionality to allow access only to explicitly configured datastores.
-
 ### Default datastore
 When a VM addresses the volume using short notation (volume_name, without @datastore), all VMs in this vmgroup will use default datastore to resolve short volume reference (volume_name will actually mean volume_name@default_datastore).
 
-If "default_datastore" is not set for a vmgroup, then datastore where the VM resides will be used as "default_datastore".
+"Default_datastore" is a required parameter when creating a vmgroup. The value is either a valid datastore name, or special string "_VM_DS.  The "Default_datastore" field is set to "_VM_DS" for "_DEFAULT" vmgroup post install and any volume create from VM which belongs to "_DEFAULT" vmgroup will be created on the datastore where VM resides.
```
Comment (shaominchen) [inline docs/features/tenancy.md]: Nits:
1. "_VM_DS - we are missing the matching quotation mark here.
2. volume create from VM => volume created from VM

File: docs/features/tenancy.md [hunk 5]
```diff
@@ -26,42 +26,56 @@ Vmgroups can be created and managed via the [Admin CLI](/user-guide/admin-cli/#V
...
+
 
 ## Example
 
-Lets consider a sample use case where there are 2 teams – Dev and Test working on Product1. Lets create separate vmgroups (namely Product1Dev and Product2Test) for each of the teams where we can put restriction on datastore consumption.
+Lets consider a sample use case where there are 2 teams – Dev and Test working on Product1. Lets create separate vmgroups (namely Product1Dev and Product1Test) for each of the teams where we can put restriction on datastore consumption.
```
Comment (shaominchen) [inline docs/features/tenancy.md]: Nit: Lets => Let's

File: docs/features/tenancy.md (hunk 4 above)
Comment (lipingxue) [inline docs/features/tenancy.md]: Done.

File: docs/features/tenancy.md (hunk 5 above)
Comment (lipingxue) [inline docs/features/tenancy.md]: Done.

**Pull request merged**